files/kubelet-wrapper.sh

```diff
@@ -18,17 +18,20 @@ mkdir --parents /etc/kubernetes
 mkdir --parents /var/lib/docker
 mkdir --parents /var/lib/kubelet
 mkdir --parents /run/kubelet
+mkdir --parents /var/log/containers
 
 exec /usr/bin/rkt run \
   --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
   --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
   --volume var-lib-docker,kind=host,source=/var/lib/docker \
   --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
+  --volume var-log,kind=host,source=/var/log,readOnly=false \
```

Comment:
Kubernetes uses and manages `/var/log/containers` to create symlinks for container logs.
We need `/var/log` to maintain `pos` for each source in fluentd. If the logging pod dies, the new pod can continue to collect logs from the last known position.